extensions/amp-analytics/analytics-vars.md

```diff
@@ -0,0 +1,105 @@
+# <a name="amp-analytics"></a> Variables supported in `amp-analytics`
+
+## Using the `vars`
+
+The format `${var-name}` can be used when a page or platform defined variable needs to be used in a request string. The `amp-analytics` tag will replace the template with the actual value at the time of construction of the analytics request. All the values substituted using this method are first url encoded using `encodeUrlComponent`. For example:
```

Comment:
If the example isn't specific to `encodeUrlComponent` I'd recommend a paragraph break, so it read more clearly that this is an example of overall usage. You could even start a new paragraph like "Here's an example of substitutions in `vars`" before the code sample.